### DISCUSSION TITLE
Libadwaita no worky

### DISCUSSION THREAD

**Discussion by xttlegendapi** [hyprwm/Hyprland#7863] [Q&A]
On both my laptop and desktop is libadwaita not working on hyprland. It looks like this:
![screenshot](https://github.com/user-attachments/assets/e4711fa6-1cc0-4f12-848c-9bc6929d4c88)

It seems to be the same as.
https://github.com/hyprwm/Hyprland/discussions/4446#discussion-6076588

somewhat relevant config things
```
env = LIBVA_DRIVER_NAME,nvidia
env = XDG_SESSION_TYPE,wayland
env = GBM_BACKEND,nvidia-drm
env = __GLX_VENDOR_LIBRARY_NAME,nvidia
env = XCURSOR_SIZE,24
env = HYPRCURSOR_SIZE,24
exec = gsettings set org.gnome.desktop.interface color-scheme "prefer-dark"
exec = gsettings set org.gnome.desktop.interface gtk-theme 'Adwaita-dark'
env = GTK_THEME,Adwaita-dark
env = QT_QPA_PLATFORMTHEME,kde # for qt apps
env = QT_QPA_PLATFORM,wayland
env = QT_STYLE_OVERRIDE,Breeze-Dark
env = QT_WAYLAND_FORCE_DPI,physical

xwayland {
    force_zero_scaling = true
}
```

**Comment by littleblack111** [comment]
im pretty sure its cuz of libadwaita its self. if ur on arch. try installing libadwaita-without-adwaita-git

see https://stopthemingmy.app/
